README.md

```diff
@@ -44,17 +44,17 @@ If you suspect a problem, please first look at the ThreatConnect connector logs
 
 ## Support
 
-1. View all API and integration offerings on the [Developer Network](https://developer.carbonblack.com/) along with reference documentation, video tutorials, and how-to guides.
-2. Use the [Developer Community Forum](https://community.carbonblack.com/t5/Developer-Relations/bd-p/developer-relations) to discuss issues and get answers from other API developers in the Carbon Black Community.
-3. Report bugs and change requests to [Carbon Black Support](http://carbonblack.com/resources/support/).
+* View all API and integration offerings on the [Developer Network](ttps://developer.carbonblack.com) along with reference documentation, video tutorials, and how-to guides.
+* Use the [Developer Community Forum](https://community.carbonblack.com/community/resources/developer-relations) to discuss issues and get answers from other API developers in the Carbon Black Community.
+* Report bugs and change requests to [Carbon Black Support](http://carbonblack.com/resources/support/).
```

Comment:
http://carbonblack.com/resources/support/ redirects to https://carbonblack.com/support/

Reply:
This is the link that was in the ticket.